test/Test/EventListenerIntrospectionTraitTest.php

```diff
@@ -193,6 +194,13 @@ public function testAssertListenerAtPriorityFailsWhenListenerIsNotFound()
                     $arguments['event'],
                     $arguments['priority']
                 ), $e->getMessage(), sprintf('Assertion failure message was unexpected: %s', $e->getMessage()));
+            // for PHPUnit 5.7
```

Comment:
ohh i missed that, is that workflow is expected, than i will change that sure =)